app_flutter/lib/state/agent.dart

```diff
@@ -58,6 +55,7 @@ class AgentState extends ChangeNotifier {
   static const String errorMessageAuthorizingAgent = 'An error occurred authorizing agent';
 
   /// Start a fixed interval loop that fetches build state updates based on [refreshRate].
+  // TODO(ianh): make this automatically trigger when listeners are added, and cancel when they're removed.
   Future<void> startFetchingStateUpdates() async {
```

Comment:
can you elaborate on this? i didn't intend to make any changes to the number of requests.

Reply:
Error on my part, just a lot of code 🎉 !  I thought `StateProvider` was called start fetching updates instead of the pages. This shouldn't be an issue since the page's will cleanup the state.

Reply:
ah ok cool.